packages/gp2-components/src/molecules/__tests__/HeaderLogo.test.tsx

```diff
@@ -0,0 +1,11 @@
+import { render } from '@testing-library/react';
+import HeaderLogo from '../HeaderLogo';
+
+describe('HeaderLogo', () => {
+  it('should render an anchor with the GP2 logo', () => {
+    const { getByRole } = render(<HeaderLogo />);
+    expect(getByRole('link').querySelector('svg > title')).toHaveTextContent(
```

Comment:
What is this lol. Try getByTitle('Logo').closest('a').href ....

Reply:
so the HeaderLogo component will render 2 logos the small and the full version, if I query for the title I'll get both, if I query for the link, since only one of them is visible I'll only get one and I make sure that in the link i have an svg with the right title

Reply:
screen.getByRole('links', { name: /GP2 Logo/i }); ?